taskfiles/dev.yml

```diff
@@ -36,6 +36,14 @@ tasks:
     status:
     - test -f '{{.VENV}}/bin/yapf'
 
+  install-go:
+    desc: install golang compiler
+    cmds:
+    - mkdir -p '{{.BUILD_ROOT}}/go/'
+    - curl -sSLf --retry 3 --retry-connrefused --retry-delay 2 'https://storage.googleapis.com/vectorizedio-public/dependencies/go1.16.5.linux-{{ARCH}}.tar.gz' | tar -xz -C '{{.BUILD_ROOT}}/go' --strip 1
+    status:
+    - test -f '{{.BUILD_ROOT}}/go/bin/go'
```

Comment:
shouldn't this be frozen at a particular version? 

nit: test if go is 1.16.5 -

Reply:
you mean creating a subfolder for each version, like `{{.BUILD_ROOT}}/go/{{.GO_VERSION}}/`?

Reply:
I pushed an update so that it it gets installed to a version-specific subfolder. Also parametrized the URL and pushed corresponding commit to vtools: https://github.com/vectorizedio/vtools/pull/191